### ISSUE TITLE
Archive Succeeded but Export failed (xcarchive is malformed)

### ISSUE THREAD

**Issue by expouic** [zoho/react-native-zohodesk-portal-sdk#5]
I am trying to build an React Native app (iOS), but after added your library I'm getting this error when generating a release build using fastlane ARCHIVE SUCCEEDED but EXPORT FAILED (xcarchive is malformed).

```
[10:49:23]: $ /usr/bin/xcrun /Library/Ruby/Gems/2.6.0/gems/fastlane-2.202.0/gym/lib/assets/wrap_xcodebuild/xcbuild-safe.sh -exportArchive -exportOptionsPlist '/var/folders/b6/zl1qvnwx6q91wz8xh_tk103r0000gn/T/gym_config20220126-67482-13efxkl.plist' -archivePath /Users/my-user/Library/Developer/Xcode/Archives/2022-01-26/test-app-zohodesk\ 2022-01-26\ 10.35.44.xcarchive -exportPath '/var/folders/b6/zl1qvnwx6q91wz8xh_tk103r0000gn/T/gym_output20220126-67482-jl4855'
+ xcodebuild -exportArchive -exportOptionsPlist /var/folders/b6/zl1qvnwx6q91wz8xh_tk103r0000gn/T/gym_config20220126-67482-13efxkl.plist -archivePath '/Users/my-user/Library/Developer/Xcode/Archives/2022-01-26/test-app-zohodesk 2022-01-26 10.35.44.xcarchive' -exportPath /var/folders/b6/zl1qvnwx6q91wz8xh_tk103r0000gn/T/gym_output20220126-67482-jl4855
error: archive at path '/Users/my-user/Library/Developer/Xcode/Archives/2022-01-26/test-app-zohodesk 2022-01-26 10.35.44.xcarchive' is malformed
** EXPORT FAILED **
[10:49:27]: Exit status: 65
```

Any idea on how to debug this in better way? 

Android build works fine.

Thanks.

**Comment by expouic** [comment]
I tried to downgrade your library version to the previous (1.0.4) and the build works fine!

**Comment by Bala-Karunakaran** [comment]
@expouic  This issue resolved in 1.0.6 version. 

Should you have any queries, feel free to hit us up at support@zohodesk.com
